fl4health/clients/basic_client.py

```diff
@@ -187,7 +221,13 @@ def train_by_epochs(
             losses = self.train_loss_meter.compute()
             loss_dict = losses.as_dict()
 
-            self._handle_logging(loss_dict, metrics, current_epoch=local_epoch, current_round=current_round)
+            # Ensure total_epochs is not None and increment
+            assert self.total_epochs is not None
+            self.total_epochs += 1
```

Comment:
If we were to unify on `self.total_steps` as our tracker, this would just be bumped into the `train_loader` loop to increment `train_steps`

Reply:
Updated in [31d94bf](https://github.com/VectorInstitute/FL4Health/pull/61/commits/31d94bf2464f27115b4672a5fae1a25a9c95e7fb)!